design-system-wordpress-plugin.php

```diff
@@ -49,6 +53,14 @@
 $content_security_policy->init();
 
 
+// Initialize the content security policy class.
+$skip_navigation = new SkipNavigation();
+$skip_navigation->init();
+
 // Initialize the enqueueing styles class.
 $enque_styles = new Style();
 $enque_styles->init();
+
+// Initialize the enqueueing scripts class.
+$enqueue_scripts = new Script();
+$enque_scripts->init();
```

Comment:
```suggestion
$enqueue_scripts->init();
```
Getting a critical error here. Can we fix the spelling of `enqueue` everywhere?